src/Web/Sfa.Das.Sas.Web/Helpers/StringUrlHelper.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Sfa.Das.Sas.Web.Helpers
+{
+    public interface IStringUrlHelper
+    {
+        string ModifyProviderNameForUrlUsage(string providerNameToProcess);
+    }
+
+    public class StringUrlHelper : IStringUrlHelper
```

Comment:
This name is a little generic, what kind of help does it offer for strings. This is also something that could be reused for things other than providers so you could call it UriEncoder and the method name EncodeTextForUri